astropy/config/configuration.py

```diff
@@ -647,7 +658,7 @@ def is_unedited_config_file(content, template_content=None):
 
 
 # this is not in __all__ because it's not intended that a user uses it
-def update_default_config(pkg, default_cfg_dir_or_fn, version=None):
+def update_default_config(pkg, default_cfg_dir_or_fn, version=None, rootname='astropy'):
```

Comment:
I did this just in case, and it might be useful in the package template. Not a clue though.